
# Do not edit this file! It is automatically generated, and any changes
# you make may be overwritten. See params_generator.m to make changes
# and regenerate the file.

model_names: [ 
  "crazy1", 
  "crazy2", 
  "crazy3", 
  "crazy4", 
  "crazy5", 
  "crazy6", 
  "crazy7", 
  "crazy8", 
  "crazy9", 
  "crazy10", 
  "crazy11", 
  "crazy12"]
/crazy1/formation_offsets: [ 1.04, 3.84, 0.00, 0.00]
/crazy2/formation_offsets: [ -0.96, 3.84, 0.00, 0.00]
/crazy3/formation_offsets: [ -2.76, 2.80, 0.00, 0.00]
/crazy4/formation_offsets: [ -3.84, 1.00, 0.00, 0.00]
/crazy5/formation_offsets: [ -3.84, -1.00, 0.00, 0.00]
/crazy6/formation_offsets: [ -2.84, -2.80, 0.00, 0.00]
/crazy7/formation_offsets: [ -1.04, -3.84, 0.00, 0.00]
/crazy8/formation_offsets: [ 0.96, -3.84, 0.00, 0.00]
/crazy9/formation_offsets: [ 2.76, -2.80, 0.00, 0.00]
/crazy10/formation_offsets: [ 3.84, -1.00, 0.00, 0.00]
/crazy11/formation_offsets: [ 3.84, 1.00, 0.00, 0.00]
/crazy12/formation_offsets: [ 2.84, 2.80, 0.00, 0.00]

robot_radius: .01
